{"ID": "59890856-e335-4a1f-bcd3-e5e4399a72b0", "effective_cve": "CVE-2017-3638", "image": {"exact": "docker.io/anchore/test_images@sha256:f56164678054e5eb59ab838367373a49df723b324617b1ba6de775749d7f91d4"}, "label": "FP", "package": {"name": "mysql", "version": "2.18.1"}, "timestamp": "2022-09-14T17:46:42+01:00", "user": "westonsteimel", "vulnerability_id": "CVE-2017-3638"}